### ISSUE TITLE
Focus and Hover Styles Should be Standardized Across the App

### ISSUE THREAD

**Issue by joekotvas** [chingu-voyages/v47-tier2-team-17#118]
**__Task Description__**
Configure focus rings and hover states for links and input fields across the site.

* All keyboard-interactive elements should display a focus ring onFocus or onFocusWithin, as appropriate.
* The focus ring should be consistently padded and visually pleasing on each element.
* When possible, something about all keyboard-interactive elements should also change color on hover.
* The accent color for interactivity should be lighter than the regular site accent color. We'll use hsl(270, 50%, 50%).
